packages/devtools-launchpad/bin/development-server.js

```diff
@@ -140,6 +143,15 @@ function handleGetConfig(req, res) {
   res.json(getConfig());
 }
 
+function handleSetConfig(req, res) {
+  const name = req.body.name;
+  const value = req.body.value;
+  setValue(name, value);
+  const output  = updateLocalConfig(root);
+  console.log(output);
+  res.end(output);
+}
```

Comment:
that's a nice idea. i was hoping to avoid this, but it looks like the webpack config bakes in the config https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/webpack.config.js#L91-L92

so it's easiest just to do this.

We can always do this in a follow up too